pkg/trace/api/otlp_test.go

```diff
@@ -2235,3 +2215,37 @@ func BenchmarkProcessRequest(b *testing.B) {
 	end <- struct{}{}
 	<-end
 }
+
+func BenchmarkProcessRequestTopLevel(b *testing.B) {
```

Comment:
Are we doing anything with the benchmark output?

Reply:
The benchmark output is just for us to compare performance across changes, since we're planning on making several changes to the OTLP receiver in the trace agent